### ISSUE TITLE
economy integration

### ISSUE THREAD

**Issue by Souperhero** [zizmax/AcuteLoot#35]
Suggestion for economy integration or addition to config.

**Comment by zizmax** [comment]
We'd like to add some sort of economy integration in a future update. Are you talking about Vault? And how were you thinking it could work in-game?

**Comment by Souperhero** [comment]
Yeah basic vault integration would be a good start. Along side the salvage feature it could add 2 ways to utilize the randomized loot in a useful way, as currently the loot only acts as the base item would normally.
